docs/guides/wifi.md

```diff
@@ -1,120 +1,52 @@
 
-## Getting the right firmware
+## Getting the firmware
 
-There are two patchsets for the linux kernel that add wifi support, but they
-each use wifi chipset firmware from different versions of macOS, and
-support different wifi chipsets.
+In order to get wifi running on Linux, we need to get the firmware from macOS and copy it to Linux with appropriate changes. Thanks to Asahi Linux, who has made the process much simpler. You may follow the steps below to get the firmware and copy it to Linux properly.
 
-Please use the table below to check which patchsets will work for your model.
+1. Boot into macOS.
 
-- You can check your Model Identifier like this:
-    - On macOS: `system_profiler SPHardwareDataType | grep "Model Identifier"`
-    - On Linux: `cat /sys/devices/virtual/dmi/id/product_name`
-- If unknown, your Mac's wifi chipset number can be found like this:
-    - On macOS: `ioreg -l | grep RequestedFiles`, the number will be
-      within the folder names, i.e. "C-`4364`\_\_s-B3" (Your model's
-      island will also be in this command's output).
-    - On Linux: `lspci -d '14e4:*'`, this will also tell you the
-      "Rev"/Revision.
+2. Open the terminal and run `cp -r /usr/share/firmware/wifi ~/Desktop`. This shall make a folder named **wifi** on the macOS Desktop.
 
-| Model Identifier | Chipset | Revision | Island   | Firmware Options     |
-|------------------|---------|----------|----------|----------------------|
-| MacBookPro16,1   | BCM4364 | 4        | Bali     | Big Sur              |
-| MacBookPro16,2   | BCM4364 | 4        | Trinidad | Big Sur              |
-| MacBookPro16,3   | BCM4377 | 4        | Tahiti   | Big Sur              |
-| MacBookPro16,4   | BCM4364 | 4        | Borneo   | Big Sur              |
-| MacBookPro15,1   | BCM4364 | 3        | Kauai    | Mojave / Big Sur     |
-| MacBookPro15,2   | BCM4364 | 3        | Maui     | Mojave / Big Sur     |
-| MacBookPro15,3   | BCM4364 | 3        | Kauai    | Mojave / Big Sur     |
-| MacBookPro15,4   | BCM4377 | 4        | Formosa  | Big Sur              |
-| MacBookAir9,1    | BCM4377 | 4        | Fiji     | Big Sur              |
-| MacBookAir8,1    | BCM4355 | 12       | Hawaii   | Mojave / Big Sur     |
-| MacBookAir8,2    | BCM4355 | 12       | Hawaii   | Mojave / Big Sur     |
-| MacMini8,1       | BCM4364 | 3        | Lanai    | Mojave / Big Sur     |
-| MacPro7,1        | BCM4364 | 4        | Kahana   | Big Sur              |
-| iMac20,1         | BCM4364 | 4        | Hanauma  | Big Sur              |
-| iMac20,2         | BCM4364 | 4        | Kure     | Big Sur              |
-| iMac19,1*        | BCM4364 | 3        | Midway   | Mojave / Big Sur     |
-| iMac19,2*        | BCM4364 | 3        | Nihau    | Mojave / Big Sur     |
-| iMacPro1,1       | BCM4364 | ?        | Ekans?   | Mojave / Big Sur?    |
+3. Copy the **wifi** folder to an external drive or a partition which is formatted to a file system readable by Linux.
```

Comment:
I don't think many people will realise they can use the EFI partition for this: `sudo diskutil mount disk0s1`, move firmware to the "EFI" disk

In future, the asahi installer will end up putting the firmware on the efi partition, and distros will eventually expect to be able to fetch it from there. Ideally we'd have the firmware always at something like /firmware.tar on the efi partition, so installer isos and so forth can automatically fetch it and get wifi working, without bundling the firmware in the iso/installers

Reply:
Well, for now the guide is written to get wifi working for T2 Macs as per the current situation. We have to option to update it when Asahi releases something, which probably is not coming too soon.

Reply:
Could you still include instructions to use the efi partition for this, as that's probably the most convenient place to use to move the firmware?

Reply:
But this shall make the things compicated for those who have a triple boot system and use a seperate EFI partition. As I said, this is a guide for the present situation, and not for when Asahi releases the installer.